programmers/난이도별/level01.문자열_내림차순으로_배치하기/HyeonJeong.py

```diff
@@ -0,0 +1,8 @@
+def solution(s):
+    small = []
+    capital = []
+    for x in s:
+        capital.append(x) if 65 <= ord(x) <= 90 else small.append(x)
+    return "".join(sorted(small + capital, reverse = True))
```

Comment:
```suggestion
    return ''.join(sorted(list(s), reverse=True))
```

사실 대소문자 구분하지 말고 그냥 정렬하면 저절로 소문자/대문자 순서로 정렬 됩니다!

Reply:
항상 부족한 부분 자세히 알려주셔서 감사합니다!!